R/utilities-output-schema.R

```diff
@@ -0,0 +1,64 @@
+#' Adds an interval to the output schema of the simulation
+#'
+#' @param simulation Simulation for which a new interval should be created
+#' @param startTime Start time of the interval in min
+#' @param endTime End time of the interval in min
+#' @param resolution Optional resolution in points/min (Default is 4 pts/hr)
+#' @param intervalName Optional Name of interval. If not specified, a unique name wil be assigned.
+#'
+#' @return Returns the interval created.
+#'
+#' @examples
+#' simPath <- system.file("extdata", "simple.pkml", package = "ospsuite")
+#'
+#' # Load the simulation
+#' sim <- loadSimulation(simPath, addToCache = FALSE, loadFromCache = FALSE)
+#'
+#' # clears the previous output schema
+#' clearIntervals(sim)
+#'
+#' # Adds a new interval starting at 1h and ending at 10h with a resolution of 10 points per hour
+#' addInterval(sim, 1 * 60, 10 * 60, 1 / 6)
+#'
+#' # Adds another interval starting at 10h and ending at 17h with a
+#' # default resolution and a specified name
+#' addInterval(sim, 10 * 60, 17 * 60, intervalName = "Second Interval")
+#' @export
+addInterval <- function(simulation, startTime, endTime, resolution = 4 / 60, intervalName = NULL) {

```

Comment:
I also suggest calling the methods `addOutputInterval` etc. in line with MoBi and PK-Sim.

Reply:
ok